tests/conftest.py

```diff
@@ -176,6 +177,19 @@ def add_service_user(sponsor=None):
     return add_service_user
 
 
+@pytest.fixture
+def add_music_org():
+    def add_music_org():
```

Comment:
@waynelai1228 I'm curious why you return a function from this fixture, rather than simply executing it.  Do the tests *need* to control when the music org is added?  Seems simpler to just have it run when a test names this fixture as a parameter, no?

Reply:
second question - is there anything about the "music" organization that is needed in the tests for which this fixture is used?  not trying to be overly pedantic, just wanting to avoid rolling too many business rules into the tests and code.  if a generic org will do, lets use that.

Reply:
Yes, you are right, I was just following the previous format and didn't notice I can run it directly.

Reply:
The music organization appeared in quite a significant amount of tests for login, I am still trying to find the best formula for this. The organization itself seems like the test-specific generic organization.